java/src/jmri/jmrix/can/cbus/CbusDccProgrammer.java

```diff
@@ -16,7 +17,7 @@
  *
  * @author Andrew Crosland Copyright (C) 2009
  */
-public class CbusDccProgrammer extends AbstractProgrammer implements CanListener {
+public class CbusDccProgrammer extends AbstractProgrammer implements CanListener, Disposable {
```

Comment:
Would it be more general to add `Disposable` to the `Programmer` interface itself, instead of to just certain subclasses?

Reply:
Sure, see latest update to this.